todo_app/data/trello_items.py

```diff
@@ -0,0 +1,66 @@
+from flask import Flask, render_template, request
+import os
+import requests
+import json
+from .TrelloClass import Item
+
+
+
+def get_trello_items():
+    url = "https://api.trello.com/1/boards/620e338747c98f4449c740c7/lists"
+
+ 
+    querystring = {
+        "key":os.getenv("TRELLO_API_KEY"),
+        "token":os.getenv("TRELLO_API_TOKEN"),
+        "cards":"open"
+    }
+    headers = {"content-type": "application/json"}
+
+    response = requests.request("GET", url, headers=headers, params=querystring)
+
+    response_json = response.json()
+
+    items = []
+
+    for trello_list in response_json:
+        for card in trello_list["cards"]:
+            card["status"] = trello_list["name"]
+            my_item = Item.from_trello_card(card, trello_list)
+            items.append(my_item)
+
+    return items
+    
+def add_card(id_List, name):
+    
+    url = "https://api.trello.com/1/cards"
+
+    querystring = {
+        "key":os.getenv("TRELLO_API_KEY"),
+        "token":os.getenv("TRELLO_API_TOKEN"),
+        "name":name,
+        "idList": id_List,
+        "idList":"620e338c101c43271f9ac494"
+    }
```

Comment:
I think you've half-converted this to a new system, but not quite finished - the below change should fix up the issue that this currently has where you aren't passing through the idList argument from `app.py`

```suggestion
def add_card(name):
    
    url = "https://api.trello.com/1/cards"

    querystring = {
        "key":os.getenv("TRELLO_API_KEY"),
        "token":os.getenv("TRELLO_API_TOKEN"),
        "name":name,
        "idList":"620e338c101c43271f9ac494"
    }
```